packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -247,6 +252,23 @@ export interface VideoBackgroundImage {
   tooltipText?: string;
 }
 
+/* @conditional-compile-remove(video-background-effects) */
+/**
+ * Contains the attibutes of a selected video background effect
+ *
+ * @beta
+ */
+export interface VideoBackgroundEffect {
+  /**
+   * Type of video background effect, can be Blur, Replacement or None.
+   */
+  type: 'Blur' | 'Replacement' | 'None';
+  /**
+   * Configuration details of selected video background effect.
+   */
+  videoEffectConfig?: BackgroundBlurConfig | BackgroundReplacementConfig;
```

Comment:
We shouldn't pass complex objects as part of state, state needs to be readonly interfaces only.

Instead here, can split `VideoBackgroundEffect` into three interfaces:
```ts
interface VideoBackgroundEffect = VideoBackgroundNoneEffect | VideoBackgroundBlurEffect | VideoBackgroundReplacementEffect;

interface VideoBackgroundNoneEffect  {
  effectName: 'None';
}

interface VideoBackgroundBlurEffect {
  effectName: 'BackgroundBlur';
}

interface VideoBackgroundReplacementEffect {
  effectName: 'BackgroundReplacement';
  url: string
}
```

Reply:
This change is just for the ClientState, the setSelectedVideoBackgroundEffect should probably still take the regaulr config object

Reply:
done